app/src/main/java/eu/depau/etchdroid/services/UsbWriteService.kt

```diff
@@ -169,11 +220,11 @@ abstract class UsbWriteService(val usbWriteName: String) : IntentService(usbWrit
             if (mWakeLock == null)
                 mWakeLock = newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, WAKELOCK_TAG)
 
-            mWakeLock.apply {
+            mWakeLock?.run {
                 if (acquire)
-                    this!!.acquire(WL_TIMEOUT /*10 minutes*/)
+                    this.acquire(WL_TIMEOUT /*10 minutes*/)
                 else
-                    this!!.release()
+                    this.release()
             }
```

Comment:
We don't use a returned value, so can just use null-safety function `run` with a lambda returned result and get rid of not-null checks